llvm/cmake/modules/FindGRPC.cmake

```diff
@@ -132,7 +132,7 @@ function(generate_protos LibraryName ProtoFile)
         ARGS ${Flags} "${ProtoSourceAbsolutePath}"
         DEPENDS "${ProtoSourceAbsolutePath}")
 
-  add_clang_library(${LibraryName} ${GeneratedProtoSource}
+  add_llvm_library(${LibraryName} ${GeneratedProtoSource}
```

Comment:
This can probably go upstream. We should not use add_clang_library in a llvm cmake module and it doesn't seem to break clangd build but I could be wrong.